drivers/can/can_mcp2515.c

```diff
@@ -618,10 +625,10 @@ static const struct mcp2515_config mcp2515_config_1 = {
 	.spi_cs_pin = DT_INST_0_MICROCHIP_MCP2515_CS_GPIOS_PIN,
 	.spi_cs_port = DT_INST_0_MICROCHIP_MCP2515_CS_GPIOS_CONTROLLER,
 #endif  /* DT_INST_0_MICROCHIP_MCP2515_CS_GPIOS_PIN */
-	.tq_sjw = CONFIG_CAN_SJW,
-	.tq_prop = CONFIG_CAN_PROP_SEG,
-	.tq_bs1 = CONFIG_CAN_PHASE_SEG1,
-	.tq_bs2 = CONFIG_CAN_PHASE_SEG2,
+	.tq_sjw = DT_INST_0_MICROCHIP_MCP2515_SJW,
+	.tq_prop = DT_INST_0_MICROCHIP_MCP2515_PROP_SEG,
+	.tq_bs1 = DT_INST_0_MICROCHIP_MCP2515_PHASE_SEG1,
```

Comment:
Similarly here for the bs1, bs2 and prop.  Less mental translation between DT and driver state.